tests/test_decode.py

```diff
@@ -344,10 +350,9 @@ def test_msg_type_17_b(self):
 
     def test_msg_type_18(self):
         msg = decode(b"!AIVDM,1,1,,A,B5NJ;PP005l4ot5Isbl03wsUkP06,0*76").asdict()
-        print(msg)
         assert msg['msg_type'] == 18
         assert msg['mmsi'] == "367430530"
-        assert msg['speed'] == 0
+        assert msg['speed'] == 0.0
         assert msg['accuracy'] == 0
         assert round(msg['lat'], 2) == 37.79
         assert round(msg['lon'], 2) == -122.27
```

Comment:
should the course (line below) be 0.0 now?

Reply:
In Python `0 == 0.0` is always `True`. So it doesn't even matter. The relevant test for ensuring that the values are floats comes a couple of lines below:

```py
assert isinstance(msg['speed'], float)
assert isinstance(msg['course'], float)
```